stockpile/roles/dmidecode/files/dmide_to_json.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
```

Comment:
Can you work to see if this works under both python 2 and python 3 environments?  RHEL 8 is moving to python 3 by default.

Reply:
Yes, good catch! Haven't verified with python3 yet.